test/print.jl

```diff
@@ -58,3 +58,13 @@ end
     d = TOML.parse(s)
     @test toml_str(d) == "user = \"me\"\n\n[julia]\n\n[option]\n"    
 end
+
+@testset "special characters" begin
+    s = """
+    "\U1f355 \0 \x0 \x1 \t \b" = "\U1f355 \0 \x0 \x1 \t \b"
+    """
+    @test roundtrip(s)
+
```

Comment:
Here's a test which the current code is failing.

```suggestion

    @test sprint(TOML.print, Dict("\x7f"=>1)) = "\"\\u007f\" = 1\n"
```